nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-controller-services.js

```diff
@@ -1090,7 +1090,7 @@
                 } else if (target.hasClass('delete-controller-service')) {
                     nfControllerService.promptToDeleteController(serviceTable, controllerServiceEntity);
                 } else if (target.hasClass('view-state-controller-service')) {
-                    nfComponentState.showState(controllerServiceEntity, controllerServiceEntity.state === 'DISABLED');
+                    nfComponentState.showState(controllerServiceEntity, controllerServiceEntity.component.state === 'DISABLED');
```

Comment:
@annanys23 I tried it and it works.  Very nice!

I was thinking maybe there should be a check for activeThreadCount as well.  We do this for [reportingTasks](https://github.com/apache/nifi/blob/rel/nifi-1.23.2/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-settings.js#L1793-L1794)

So we would have something like:
```
var canClear = controllerServiceEntity.status.runStatus === 'DISABLED' && (nfCommon.isUndefinedOrNull(controllerServiceEntity.status.activeThreadCount) || controllerServiceEntity.status.activeThreadCount === 0);
nfComponentState.showState(controllerServiceEntity, canClear);
```

Reply:
Thanks for testing!  Controller services, unlike reporting tasks and processors do not have activeThreads.  When a controller service is disabled, all connected components are stopped.  So, activeThreadCount for those should be 0.